src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,21 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
```

Comment:
It's better to write in this way (because it's actually list):
```suggestion
  <ul className="TodoList">
```

just FYI (for the future)